core/src/test/scala/org/apache/spark/scheduler/SparkListenerSuite.scala

```diff
@@ -255,21 +262,19 @@ class SparkListenerSuite extends SparkFunSuite with LocalSparkContext with Match
         taskInfoMetrics.map(_._2.executorDeserializeTime),
         stageInfo + " executorDeserializeTime")
 
-      /* Test is disabled (SEE SPARK-2208)
       if (stageInfo.rddInfos.exists(_.name == d4.name)) {
         checkNonZeroAvg(
           taskInfoMetrics.map(_._2.shuffleReadMetrics.get.fetchWaitTime),
           stageInfo + " fetchWaitTime")
       }
-      */
 
       taskInfoMetrics.foreach { case (taskInfo, taskMetrics) =>
-        taskMetrics.resultSize should be > (0L)
+        taskMetrics.resultSize should be > 0L
         if (stageInfo.rddInfos.exists(info => info.name == d2.name || info.name == d3.name)) {
-          taskMetrics.inputMetrics should not be ('defined)
-          taskMetrics.outputMetrics should not be ('defined)
+          taskMetrics.inputMetrics should not be 'defined
+          taskMetrics.outputMetrics should not be 'defined
           taskMetrics.shuffleWriteMetrics should be ('defined)
-          taskMetrics.shuffleWriteMetrics.get.bytesWritten should be > (0L)
+          taskMetrics.shuffleWriteMetrics.get.bytesWritten should be > 0L
```

Comment:
Ah yeah, that's because I usually write:
taskMetrics.shuffleWriteMetrics shouldBe 'defined
instead of:
taskMetrics.shuffleWriteMetrics should be ('defined)
I find it much quicker to write and Idea doesn't warn me on the useless brackets. Never mind I reverted this.

I just pushed the 25ms version. "Jenkins retest this please" :stuck_out_tongue_winking_eye: